src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/MicroprofileSpec.java

```diff
@@ -30,39 +30,49 @@
 public enum MicroprofileSpec {
     // @formatter:off
     CONFIG("config", "Config",
+            "Configuration - externalize and manage your configuration parameters outside your microservices",
             Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14,
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22))
     , FAULT_TOLERANCE("fault_tolerance", "Fault Tolerance",
+            "Fault Tolerance - all about bulkheads, timeouts, circuit breakers, retries, etc. for your microservices",
             Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14,
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22))
     , JWT_AUTH("JWT_auth", "JWT Auth",
+            "JWT Propagation - manage and propagate security across your microservices",
             Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14,
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22))
     , METRICS("metrics", "Metrics",
+            "Metrics - Gather and create operational and business measurements for your microservices",
             Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14,
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22))
     , HEALTH_CHECKS("health_checks", "Health Checks",
+            "Health Checks - ensure your microservices is up and running by enabling health checks",
```

Comment:
Health check is more than just is it up and running

Health Checks - Verify the health of your microservices with custom verifications

Reply:
Addressed.